src/pages/LogInWithShortLivedTokenPage.js

```diff
@@ -55,38 +56,29 @@ class LogInWithShortLivedTokenPage extends Component {
 
         const isUserSignedIn = this.props.session && this.props.session.authToken;
         if (!isUserSignedIn) {
+            Log.info('[LoginWithShortLivedTokenPage] User not signed in - signing in with short lived token');
             Session.signInWithShortLivedToken(accountID, email, shortLivedToken);
             return;
         }
 
-        this.signOutIfNeeded(email);
-    }
-
-    componentDidUpdate() {
-        if (!lodashGet(this.props, 'session.authToken', null)) {
+        if (this.signOutIfNeeded(email)) {
             return;
```

Comment:
Want to put a log line in here to log the case where a different user had to be logged out?

Reply:
There's one on ln 101

Reply:
That logic doesn't need to be in a method anymore tho

Reply:
Gonna just leave it I think